### PR TITLE
Feature/188 generate UUID for groupid

### LINKED ISSUE #188: Generate UUID for GroupId
## Feature Title

add a feature to create uuid for groupid.

## Acceptance Criteria

- [ ] Criterion 1
- [ ] Criterion 2

## Technical Details

- API Changes:
- Database Changes:
- Dependencies:
- Tests

## Additional Context

Any other internal context or references.

### PR DESCRIPTION
**Pull request by taniguchiiqqq**
## Overviews of implementation
make groupId UUID

### CODE REVIEW

**Comment by taniguchiiqqq** [comment]
@hitohata 
Please review this one after https://github.com/genesis-tech-tribe/nishiki-backend/pull/191 PR, because the check-type-document failure is fixed in it.
Sorry for bothering you.

File: backend/main/src/Group/Domain/Entities/Group.ts [hunk 1]
```diff
@@ -155,13 +156,26 @@ export class Group extends AggregateRoot<string, IGroupProps> {
...
 export class GroupId extends Identifier<string> {
-	static create(id?: string): Result<GroupId, GroupIdDomainError> {
-		if (id) {
-			return Ok(new GroupId(id));
+	/**
+	 * the group ID is UUID.
+	 * @param id
+	 */
+	static create(id: string): Result<GroupId, GroupIdDomainError> {
+		if (!uuidValidate(id)) {
```
Comment (hitohata) [inline backend/main/src/Group/Domain/Entities/Group.ts]: Would you mind if I ask you to merge the develop branch and use the following function? Its function also checks if the argument is valid UUID V4.

https://github.com/genesis-tech-tribe/nishiki-backend/blob/develop/backend/main/src/Shared/Utils/Validator.ts

**Review by hitohata — APPROVED** [review]
LGTM. Sorry for the late reply.

File: backend/main/src/Group/Domain/Entities/Group.ts (hunk 1 above)
Comment (taniguchiiqqq) [reply backend/main/src/Group/Domain/Entities/Group.ts]: @hitohata 
I see! I'll do that before merging

**Pull request merged**